src/lzqgame.ts

```diff
@@ -477,8 +479,8 @@ async function playerMakeMove(
                 moves: [...moveHistory, pendingMove],
             });
             myGame = await getGame(room);
-            
-          if (!myGame) {
+
+            if (!myGame) {
```

Comment:
more weird formatting

Reply:
It was previously too far left by one tab, I think? Auto formatter fixed this.